azure-monitor-opentelemetry-distro/setup.py

```diff
@@ -1,21 +1,103 @@
+#!/usr/bin/env python
+
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License in the project root for
+# Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
 
+
 import os
+import re
 
-import setuptools
+from setuptools import find_packages, setup
 
-BASE_DIR = os.path.dirname(__file__)
-VERSION_FILENAME = os.path.join(
-    BASE_DIR, "src", "azure", "monitor", "opentelemetry", "distro", "version.py"
-)
-PACKAGE_INFO = {}
-with open(VERSION_FILENAME, encoding="utf-8") as f:
-    exec(f.read(), PACKAGE_INFO)
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-monitor-opentelemetry-distro"
+PACKAGE_PPRINT_NAME = "Azure Monitor Opentelemetry Distro"
```

Comment:
print typo

Reply:
`PPRINT` I believe stands for "pretty print" so that is actually intentional. https://realpython.com/lessons/pretty-print/